.gitignore

```diff
@@ -76,7 +76,7 @@ target/
 .ipynb_checkpoints/
 
 # exclude data from source control by default
-/data/
+data
```

Comment:
That is excluding all code that have /data/ in path: for example habitat/internal/data, test/data. If baselines will have data preparation code and put it in data folder inside of baselines it will be ignored. It looks too general to me and we shouldn't spread data around code base if it's not checked in or not in root data folder.

Reply:
My problem here is this does not work for symlinks. I looked online and it seems the only way to ignore symlinks is to not use slashes. It's generally bad practice to have your dataset inside your project folder anyways (causes you to need multiple copies). I don't know what the right solution is here.

Reply:
Ok, let's remove slashes.